app/src/main/res/layout/activity_login_screen.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It's amazing that you are trying out ConstraintLayout. However, I'll advice that you become proficient in the use of Frame, Linear and Relative layouts first before progressing to ConstraintLayouts.

Reply:
I have switched to linear layout noted.